docs/merkletree/treetex/main.go

```diff
@@ -247,16 +244,14 @@ func renderTree(prefix string, treeSize, index int64) {
 }
 
 // nodeKey returns a stable node identifier for the passed in node coordinate.
-func nodeKey(height, index int64) string {
-	return fmt.Sprintf("%d.%d", height, index)
+func nodeKey(level, index int64) string {
+	return fmt.Sprintf("%d.%d", level, index)
 }
 
-// toNodeKey converts a storage.NodeID to the corresponding stable node
+// toNodeKey converts a compact.NodeID to the corresponding stable node
 // identifier used by this tool.
-func toNodeKey(n storage.NodeID) string {
-	d := int64(maxLen - n.PrefixLenBits)
-	i := n.BigInt().Int64() >> uint(d)
-	return nodeKey(d, i)
+func toNodeKey(n compact.NodeID) string {
```

Comment:
Can it not just use the `NodeID` directly?

Reply:
Great idea. Done.